src/Magnum/Math/Test/ColorTest.cpp

```diff
@@ -850,6 +854,30 @@ void ColorTest::xyY() {
     CORRADE_COMPARE(xyYToXyz(xyY), xyz);
 }
 
+void ColorTest::strictWeakOrdering() {
+    StrictWeakOrderingFunc o;
+
+    auto r = Color4::red();
+    auto g = Color4::green();
+    auto b = Color4::blue();
+
+    CORRADE_VERIFY(o(b, r));
+    CORRADE_VERIFY(o(g, r));
+    CORRADE_VERIFY(o(b, g));
+
+    auto ba = b;
+    ba.a() = 0.5;
```

Comment:
`0.5f`, since it's a `float` literal. I *think* some compilers are emitting a warning here, but not sure which.